data/hai/unfettered 0 prologue.txt

```diff
@@ -516,57 +521,992 @@ mission "Unfettered Jump Drive Trading"
+	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."
+
+# SPOILER: he tests your military aptitude before asking you to take him to pirate space.
+# gameplay wise if you win this mission you should be able to do the following ones, it tests if your strength is high enough.
+# Since it is the first one you can do it many times until you succeed.
+mission "<uhai> 2: Prove Your Strength"
+	autosave
+	repeat
+	deadline 2
+	name `Defeat the captain`
+	description `A young Unfettered captain has provoked you into a duel. Disable his ship and then land back on <destination>. You only have one day, but this duel can happen multiple times.`
+	source "Darkcloak"
+	to offer
+		or
+			has "<uhai> 2: Prove Your Strength: deferred"
+			has "Wanderers: Defend Vara Ke'sok: aborted"
+			random < "Unfettered Jump Drive 1: offered" * 20 + "Unfettered Jump Drive 2: offered" * 25 + "Unfettered Jump Drive 3: offered" * 30 + "Unfettered Jump Drive 4: failed" * 25
+		# 3 is what a default built Sea Scorpion has
+		"armament deterrence" >= 3
+		not "<uhai> 2: Prove Your Strength: done"
+	on offer
+		conversation
+			branch waiting
+				or
+					has "<uhai> 2: Prove Your Strength: deferred"
+					has "<uhai> 2: Prove Your Strength: offered"
+			`You start wandering in the spaceport, and on the way you notice just how much everything around you here is simple, efficient, and ordered, unlike the world of Firelode. The Unfettered present are proudly showing their weapons, more so than on other planets, and some of them are even armored. They seem ready for action, at any time. Thankfully they no longer consider you an enemy, at least for the moment.`
+			`	Eventually your attention is drawn to a gathering, where someone is speaking loudly to a small crowd of people. They all seem very familiar with each other, and as you watch, they have noticed you too. The one who was addressing the others saunters through the group and over to you. This could mean trouble.`
+			`	"Arrr, be this the bilge-sucking <, unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`
+			choice
+				`	"I fear nothing. Do you?"`
+					goto provoke
+				`	(Say nothing.)`
+				`	(Run away.)`
+					goto run
+			`	"See? The lily-livered monkey dares not even speak! This scallywag be no more than shark bait! Even our 'umble clipper would be sending ye straight to Davy Jones' locker! We be ready to prove it, too, yet we dare not, lest we lose our dancing monkey to the briny deep! But mateys, there be no shame if it dance away, with a dinghy looking like that!" He seems very full of himself. Perhaps someone should teach him a lesson.`
```

Comment:
He's very deliberately talking in human references instead of Hai.

The whole pirate speak thing is a deliberately worn affectation and that wouldn't be nearly as clear if it were adapted to Hai cultural references, especially before he explains that himself.

Reply:
Ah, I haven't gotten to that part yet.